app/src/main/java/com/sunasterisk/movieintroduction/ui/main/GetStarted.kt

```diff
@@ -0,0 +1,25 @@
+package com.sunasterisk.movieintroduction.ui.main
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.animation.AnimationUtils
+import com.sunasterisk.movieintroduction.R
+import kotlinx.android.synthetic.main.activity_get_started.*
+
+class GetStarted : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_get_started)
+
+        val anim = AnimationUtils.loadAnimation(applicationContext, R.anim.fade_in)
+        buttonGetStarted.startAnimation(anim)
+
+        buttonGetStarted.setOnClickListener {
+            var intent = Intent(this, MainActivity::class.java)
+            startActivity(intent)
```

Comment:
conventions 2.11 
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md
Tạo 1 hàm getIntent(..) trong MainActivity 
rồi ở đây chỉ gọi 
startActivity(MainActivity.getIntent(this))